react/src/Mpool.js

```diff
@@ -44,32 +46,28 @@ function MpoolMessage(props) {
     const i = props.i
     const msg = props.msg
 
+    const {data} = useQuery(EpochQuery)
+    let currEpoch = data.epoch.Epoch+''
+    console.log(currEpoch)
+    let sentEpoch = msg.SentEpoch+''
+    console.log(sentEpoch)
+    let t = (currEpoch - sentEpoch)*data.epoch.SecondsPerEpoch+''
+
     const handleParamsClick = (el) => {
         el.target.classList.toggle('expanded')
     }
 
-    let t = msg.ElapsedSeconds+''
     let hh = Math.floor(t / 3600)
     t %= 3600
     let mm = Math.floor(t / 60)
-    let etime = `${hh} hours and ${mm} minutes`
+    t %= 60
+    let etime = `${hh} hours and ${mm} minutes ${t} seconds`
+    let epochInfo = `${sentEpoch} (${currEpoch - sentEpoch} epochs / ${etime} ago)`
```

Comment:
You can use the moment library to do most of this work:
```
let elapsed = data.epoch.Epoch - msg.SentEpoch
moment().add(-elapsed*30, 'seconds').fromNow()
```

Reply:
I have updated the code to use moment library. I was thinking about `Show All Messages` and it does not seems to be adding any value. It is not helpful for debugging either. Plus, it is slow to load.
I would vote to remove the section all together and use that section to show alert messages. We call the button `show alerts` and render full messages there. In a full size SP, a lot of messages are pending at a time and this would be helpful. Or get rid of the button completely.

Reply:
Agree, we don't care about `Show All Messages` and we should remove it.